src/components/TopNav.js

```diff
@@ -80,7 +80,7 @@ export default class TopNav extends Component {
   render () {
     const {isOpen} = this.state;
     return (
-      <Headroom disableInlineStyles calcHeightOnResize>
+      <Headroom disableInlineStyles calcHeightOnResize upTolerance={100}>
```

Comment:
I notice that in iPhone Safari there glitch when scrolling up. The top of the first card is hidden underneath the navbar and only after some delay fully reveals. Is this maybe headroom related?

![dracor-capture](https://user-images.githubusercontent.com/3439037/95600882-06f1b800-0a53-11eb-9dd1-4e81f961a297.gif)

Reply:
There are indeed multiple small issues regarding height calculation by headroom. Thank you for noticing that! I'll see whether it can be solved in the future. Maybe we should even look for an alternative package.